### DISCUSSION TITLE
"Save As..." to choose file save location?

### DISCUSSION THREAD

**Discussion by JeremyNguyenPhD** [aclap-dev/video-downloadhelper#1549] [Help and Q&A] [answered]
### Your question

Is there any way that we can choose the location for the videos we're downloading for each file?

I do realise that in Settings we can set the default directory that videos are saved to, but I'd like to be able to choose each time I download.

Is that possible? Or can I put that in as a feature request?

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"1b5bea6eba484fae"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "/Users/jeremy/dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "[]"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 27
}

**Comment by jcfrog** [comment, marked as answer]
Hi,
You can use the "Download as" and even choose it as your default action. 
![CleanShot 2024-12-16 at 09 46 12@2x](https://github.com/user-attachments/assets/ce55a8ea-7dbf-4207-9a81-56ce7829d8f8)

**Reply by JeremyNguyenPhD** [reply]
Thank you so much!

**Comment by GrampaWildWilly** [comment]
You can read #665 & #814 to get a more thorough explanation of how it all works.
